src/execution_plan/ops/op_delete.c

```diff
@@ -25,17 +26,9 @@ OpBase* NewDeleteOp(AST_DeleteNode *ast_delete_node, QueryGraph *qg, Graph *g, R
     op_delete->qg = qg;
```

Comment:
Regarding line 20 (which I can't add a comment on directly):

I don't think `_LocateEntities` requires a forward declaration anymore; you can just move its definition to that spot.

Reply:
You're right, but to be consistent with other operation implementation, I prefer the New function to be the first function in these files.